From 2d4b5e248cbcc3ebe64577dc29cf8756926ddf7e Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Wed, 25 Jan 2012 11:26:55 -0500
Subject: [PATCH] libMirage: configure: do not check for gtk-doc.pc

GTK_DOC_CHECK is sufficient. The unusual gtk-doc.pc-based check
needlessly prevents libmirage from configuring with --disable-gtk-doc
(which is the default) if the full gtk-doc package is not installed.
---
 libmirage/configure.ac |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/libmirage/configure.ac b/libmirage/configure.ac
index e74b0a0..3224f59 100644
--- a/libmirage/configure.ac
+++ b/libmirage/configure.ac
@@ -57,12 +57,6 @@ AC_HEADER_STDC
 AC_SYS_LARGEFILE
 
 # Gtk-doc check
-PKG_CHECK_EXISTS(gtk-doc >= 1.4, [have_gtkdoc="yes"], [have_gtkdoc="no"])
-AC_MSG_CHECKING([for gtk-doc])
-AC_MSG_RESULT(${have_gtkdoc})
-if test ${have_gtkdoc} = "no"; then 
-    AC_MSG_ERROR("*** gtk-doc not found ***")
-fi
 GTK_DOC_CHECK([1.4])
 
 # Libraries check
-- 
1.7.8.4

